src/back/hlsl/writer.rs

```diff
@@ -874,7 +874,7 @@ impl<'a, W: fmt::Write> super::Writer<'a, W> {
             }
             _ => {
                 return Err(Error::Unimplemented(format!(
-                    "write_value_type {:?}",
+                    "back::hlsl::Writer::write_value_type {:?}",
```

Comment:
is this necessary?

Reply:
I added this because the other back end generated an identical error message, and I ended up jumping to the conclusion that I was looking at the wrong code, which befuddled me for a few minutes.

Reply:
Where did you see the error? Is it from snapshots?
I just don't think this scales well. We don't want to have long prefixes for each and every error in each backend.
Instead, the `snapshot.rs` could just `println!("Testing MSL")`, and that's it.

Reply:
Okay, that's a good alternative.

Reply:
I've implemented this in #1427. I'll remove the change here.